tests/db-utils.ts

```diff
@@ -120,15 +120,42 @@ export async function cleanupDb(prisma: PrismaClient) {
 		{ name: string }[]
 	>`SELECT name FROM sqlite_master WHERE type='table' AND name NOT LIKE 'sqlite_%' AND name NOT LIKE '_prisma_migrations';`
 
+	const migrationPaths = fs
+		.readdirSync('prisma/migrations')
+		.filter((dir) => dir !== 'migration_lock.toml')
+		.map((dir) => `prisma/migrations/${dir}/migration.sql`)
+
+	const migrations = migrationPaths.map((path) => {
+		// Parse the sql into individual statements
+		const sql = fs
+			.readFileSync(path)
+			.toString()
+			.split(';')
+			.map((statement) => (statement += ';'))
+
+		// Remove empty last line
+		sql.pop()
```

Comment:
This should take care of any empty lines.

```suggestion
			.map(statement => statement.trim())
			.filter(Boolean)
			.map((statement) => `${statement};`))
```